packages/cli/src/commands/generate.ts

```diff
@@ -345,10 +342,7 @@ async function getContract({
       typeof resolvedAddress === 'string'
         ? JSON.stringify(resolvedAddress)
         : // Remove quotes from chain id key
-          JSON.stringify(resolvedAddress, null, 2).replace(
-            /^\s*"(\d)":/gm,
-            '$1:',
-          )
+          JSON.stringify(resolvedAddress, null, 2).replace(/"(\d*)":/gm, '$1:')
```

Comment:
regex i used before only worked for single digits 🤦‍♂️

this one is much better: https://regexr.com/7700l